simpleSequential/test_isTwoStringEqual.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from isTwoStringEqual import isTwoStringEqual
+
+class TestIsTwoStringEqual(unittest.TestCase):
```

Comment:
Wrong tests. Read the function name “isTwoStringEqual”. I did not see any string in the test.